packages/commands/src/query/chains/show.ts

```
@@ -0,0 +1,268 @@
+// Helper function to convert field names to lowercase with underscores
+const toFieldName = (str: string): string => {
+  return str.toLowerCase().replace(/[^a-z0-9]/g, "_");
+};
+
+// Helper function to get field value from chain or derived data
+const getFieldValue = (
+  chain: ObserverSupportedChain,
+  field: string,
+  tokens: string[],
+  confirmations: string | undefined,
+  viemChain: any
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)